### DISCUSSION TITLE
Mysql & mysqldump not found error in admin portal after Installing Easy Backup Bundle in Kimai Docker Setup

### DISCUSSION THREAD

**Discussion by kcommerce** [kimai/kimai#5046] [Plugins]
We have installed Kimai version 2.20 using Docker Compose in production mode (refer to the YAML file below).
Afterward, we installed the Easy Backup Bundle following the instructions provided [here](https://www.kimai.org/store/maximiliangross-easy-backup-bundle.html).

However, we encountered an issue where mysql and mysqldump are not available in the web container (kimai/kimai2:apache). (Please see the warning in attached picture.)

Since there are two containers—one for the web and another for the database—how should we proceed with the installation? Your guidance would be greatly appreciated.

  ```bash

adminproject@projectmgt:~$ docker container list
CONTAINER ID   IMAGE                 COMMAND                  CREATED      STATUS                PORTS                                               NAMES
46addb5070d3   kimai/kimai2:apache   "docker-php-entrypoi…"   3 days ago   Up 3 days (healthy)   80/tcp, 0.0.0.0:8001->8001/tcp, :::8001->8001/tcp   kimai-kimai-1
d97e3f2530d8   mysql:8.3             "docker-entrypoint.s…"   3 days ago   Up 3 days (healthy)   3306/tcp, 33060/tcp                                 kimai-sqldb-1


  ```

  ```bash
adminproject@projectmgt:~$ cd kimai/

adminproject@projectmgt:~/kimai$ cat docker-compose.yaml
version: '3.5'
services:

  sqldb:
    image: mysql:8.3
    volumes:
      - mysql:/var/lib/mysql
    environment:
      - MYSQL_DATABASE=kimai
      - MYSQL_USER=kimaiuser
      - MYSQL_PASSWORD=kimaipassword
      - MYSQL_ROOT_PASSWORD=changemeplease
    command: --default-storage-engine innodb
    restart: unless-stopped
    healthcheck:
      test: mysqladmin -p$$MYSQL_ROOT_PASSWORD ping -h localhost
      interval: 20s
      start_period: 10s
      timeout: 10s
      retries: 3

  kimai:
    image: kimai/kimai2:apache
    volumes:
      - data:/opt/kimai/var/data
    ports:
      - 8001:8001
    environment:
      - ADMINMAIL=admin@kimai.local
      - ADMINPASS=changemeplease
      - "DATABASE_URL=mysql://kimaiuser:kimaipassword@sqldb/kimai?charset=utf8mb4&serverVersion=8.3.0"
      - TRUSTED_HOSTS=nginx,localhost,127.0.0.1,192.168.10.185
    restart: unless-stopped

volumes:
  data:
  mysql:

  ```
![kimai-mysql-problem-ticket](https://github.com/user-attachments/assets/ceb0c057-84cd-4cd8-9846-0b502d599408)

**Comment by Fellurion** [comment]
"DATABASE_URL=mysql://kimaiuser:kimaipassword@sqldb/kimai?charset=utf8mb4&serverVersion=8.3.0"


Ur set the right User and Password for Ur Database? 
actually u set kimaiuser as user and kimaipassword as password for login to Ur database 


And this ? 
ADMINPASS=changemeplease

First try too See Ur compose 

https://www.kimai.org/documentation/docker-compose.html

after Show other docu parts. Ur database is Not correct

**Comment by kevinpapst** [comment]
You should post that question in the EasyBackup repository.

Short answer: I don't think this is possible with the default image and the way EasyBackup currently works. EasyBackup expects the mysql/dump binaries on the local system and the Kimai default image does not include those.

**Comment by nosi60** [comment]
You should install mariadb-client - its working to me...
apt-get install mariadb-client

![grafik](https://github.com/user-attachments/assets/5419d8d8-c027-4955-ae28-923d9e5868f2)

However, a direct restore is not possible - but this way you have all the data in zipfile.
